_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
+                                <span class="glyphicon glyphicon-comment" aria-hidden="true"></span>
+                            </a>`;
+            }
+            if (data[i].hasOwnProperty("domain") && data[i]['domain'] !== undefined) {
+                domainInner[0] = data[i]['domain'];
+            }
+            if (description !== undefined && description.toString().length > 140) {
+                description = description.toString().slice(0, 140) + '...'
+            }
+            if (data[i]["contact"]) {
+                contact =`
+                        <a class="btn btn-default btn-sm" href="mailto:${data[i]["contact"]["email"]}" aria-label="Send an email to ${title}"
+                           title="Send an email to ${title}">\
+                            <span class="glyphicon glyphicon-envelope" aria-hidden="true"></span>
+                        </a>`;
+            }
+            if (data[i]["publications"] && data[i]["publications"].length > 0) {
+                publication = `
+                            <a class="btn btn-default btn-sm" href="${data[i]["publications"][0]["id"]}" aria-label="View the primary publication for ${title}" title="View the primary publication for ${title}">
+                                <span class="glyphicon glyphicon-book" aria-hidden="true"></span>
+                            </a>`;
+            }
+            if (data[i]["in_foundry_order"]) {
+                foundry_order = `
+                        <a class="col-sm-1 btn btn-default btn-sm" href="/principles/fp-000-summary.html" aria-label="View the OBO Foundry criteria for the Foundry status of ${title}" title="View the OBO Foundry criteria for the Foundry status of ${title}">
+                               <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                        </a>             
+                `;
+            }
+            if (activity_status === "inactive") {
+                is_inactive = "inactive_row";
+            }
+            if (data[i]["is_obsolete"]) {
+                is_obsolete = "(obsolete)"
+            }
+            // table row template
+            let template = `
+                <tr class="row ${is_inactive}">
+                    <td class="col-sm-1">
+                        <a class="" href="ontology/${id}.html">
+                           ${id} 
+                        </a>
+                        <span style="background-color: #ff8d82">${is_obsolete}</span>    
+                    </td>
+                    <td class="col-sm-1">
+                        ${title}
+                    </td>
+                    <td class="col-sm-3">
+                        ${description}
+                        <small>
+                            <a href="ontology/${id}.html">Detail</a>
+                        </small>
+                    </td>
+                    <td class="col-sm-5">
+                        <div class="ic-display">
```

Comment:
Group all buttons together, reclaim countless horizontal and vertical space
```suggestion
                        <div class="btn-group" role="group" aria-label="Quick navigation">
```

Results:
<img width="1207" alt="Screen Shot 2022-08-01 at 00 20 29" src="https://user-images.githubusercontent.com/5069736/182047568-db0d1088-8907-429c-87dd-42ef443e2580.png">